File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        left ,right = 0,1
        while  left <= len(nums)-1:
            if right <= len(nums)-1:
                if nums[left]+nums[right] == target:
                    return [left,right]
                right += 1
            else:
                left += 1
                right = left + 1
            


        

File: 1-two-sum/two-sum.ts
function twoSum(nums: number[], target: number): number[] {
    let left = 0
    let right = 1

    while (left < right) {
        if (nums[left] + nums[right] === target) {
            return [left, right]
        }


        if (right === nums.length - 1) {
            left++
            right = left + 1
        } else {
            right++

        }

    }
};